state-chain/custom-rpc/src/lib.rs

```diff
@@ -518,6 +526,9 @@ pub trait CustomApi {
 		side: Order,
 	);
 
+	#[subscription(name = "subscribe_scheduled_swaps", item = Vec<AssetAmount>)]
```

Comment:
Shouldn't the items here be `Vec<ScheduledSwap>`? else we can't know the direction

Reply:
Yeah. I copy pasted from somewhere. Not sure what this actually affects, since the stream does correctly return `ScheduledSwap` items in practice.